advanced_bank_reconciliation/advanced_bank_reconciliation/doctype/advance_bank_reconciliation_tool/tests/test_partial_allocation.py

```
@@ -489,3 +492,678 @@ def test_normalise_helper_cascade_negative_credit_note(self):
+		si = create_test_sales_invoice(outstanding=200)
+		pe = get_payment_entry("Sales Invoice", si.name)
+		pe.paid_to = TEST_BANK_GL_ACCOUNT
+		pe.paid_amount = 200
+		pe.received_amount = 200
+		pe.reference_no = "_ABR-MULTI"
+		pe.reference_date = nowdate()
+		pe.posting_date = nowdate()
+		try:
+			pe.insert(ignore_permissions=True)
+			pe.submit()
+		except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)